lambda_2/lambda_function.py

```diff
@@ -0,0 +1,52 @@
+import json
+import csv
+import boto3
+import os
+import pandas as pd
+import awswrangler as wr
```

Comment:
Since you are using wrangler and pandas, youll need a lambda layer. Id recommend just using the public aws one available "Aws Sdk for pandas"

Reply:
yep, located the public arn and built a layer into the cdk.